masterlist.yaml

```
@@ -523,6 +523,35 @@ plugins:
       - 'Order_Of_Sanctity.ESP'
       - 'Walkers of Morrowind.esp'
 
+###### Locations - Overhauls ######
+  ###Beautiful cities of Morrowind###
+  - name: 'Beautiful cities of Morrowind.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after:
+      - 'Better Landscapes Stonewood Pass (RP Edit).esp'
+      - 'Morrowind Anti-Cheese.esp'
+
+  - name: 'BOM_pathgrid_reset.esp'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after:
+      - name: 'Beautiful cities of Morrowind.ESP'
+      - name: 'multipatch.esp'
+
+  - name: 'BOM_OpenMW_plazas.esp'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after:
+      - name: 'Beautiful cities of Morrowind.ESP'
+      - name: 'multipatch.esp'
+      - name: 'BOM_pathgrid_reset.esp'
+
+  - name: 'Interior exterior flag reset.esp'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after:
+      - name: 'Beautiful cities of Morrowind.ESP'
+      - name: 'multipatch.esp'
+      - name: 'BOM_pathgrid_reset.esp'
+      - name: 'BOM_OpenMW_plazas.esp'
```

We'll only add spaces if the files are from different mods.

```suggestion
  - name: 'Beautiful cities of Morrowind.ESP'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after:
      - 'Better Landscapes Stonewood Pass (RP Edit).esp'
      - 'Morrowind Anti-Cheese.esp'
  - name: 'BOM_pathgrid_reset.esp'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after:
      - name: 'Beautiful cities of Morrowind.ESP'
      - name: 'multipatch.esp'
  - name: 'BOM_OpenMW_plazas.esp'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after:
      - name: 'Beautiful cities of Morrowind.ESP'
      - name: 'multipatch.esp'
      - name: 'BOM_pathgrid_reset.esp'
  - name: 'Interior exterior flag reset.esp'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after:
      - name: 'Beautiful cities of Morrowind.ESP'
      - name: 'multipatch.esp'
      - name: 'BOM_pathgrid_reset.esp'
      - name: 'BOM_OpenMW_plazas.esp'
```